index.js

```diff
@@ -400,10 +400,7 @@ RingPop.prototype.protocolJoin = function protocolJoin(options, callback) {
     this.serverRate.mark();
     this.totalRate.mark();
 
-    this.membership.addMember({
-        address: joinerAddress,
-        incarnationNumber: options.incarnationNumber
-    });
+    this.membership.makeJoin(joinerAddress, options.incarnationNumber);
```

Comment:
Incidental refactor to be more consistent with `makeFaulty`, `makeSuspect`, etc operations